vitest.config.ts

```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from 'vitest/config';
+import react from '@vitejs/plugin-react';
+import { resolve } from 'path';
+
+export default defineConfig({
+  plugins: [react()],
+  test: {
+    environment: 'jsdom',
+  },
+  resolve: {
+    alias: {
+      '~/': `${resolve(__dirname, 'src')}/`,
+    },
+  },
+});
```

Comment:
i think this dependency and this file should be removed. `vitest` should be replaced with `jest` or `cypress` what do you think?

Reply:
https://github.com/defi-wonderland/web3-react-boilerplate/commit/9a9683fc0ce55ee0a3c687f7ac80783cade96b46